tests/integration/golang/aim/experiment/get_experiment_test.go

```diff
@@ -0,0 +1,111 @@
+//go:build integration
+
+package experiment
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetExperimentTestSuite struct {
+	suite.Suite
+	client   *helpers.HttpClient
+	fixtures *fixtures.ExperimentFixtures
+}
+
+func TestGetExperimentTestSuite(t *testing.T) {
+	suite.Run(t, new(GetExperimentTestSuite))
+}
+
+func (s *GetExperimentTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	fixtures, err := fixtures.NewExperimentFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.fixtures = fixtures
+}
+
+func (s *GetExperimentTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.fixtures.UnloadFixtures())
+	}()
+	experiment, err := s.fixtures.CreateExperiment(context.Background(), &models.Experiment{
+		Name: "Test Experiment",
+		Tags: []models.ExperimentTag{
+			{
+				Key:   "key1",
+				Value: "value1",
+			},
+		},
+		CreationTime: sql.NullInt64{
+			Int64: time.Now().UTC().UnixMilli(),
+			Valid: true,
+		},
+		LastUpdateTime: sql.NullInt64{
+			Int64: time.Now().UTC().UnixMilli(),
+			Valid: true,
+		},
+		LifecycleStage:   models.LifecycleStageActive,
+		ArtifactLocation: "/artifact/location",
+	})
+	assert.Nil(s.T(), err)
+
+	var resp response.GetExperiment
+	err = s.client.DoGetRequest(
+		fmt.Sprintf(
+			"/experiments/%d", *experiment.ID,
+		),
+		&resp,
+	)
+	assert.Nil(s.T(), err)
+
+	assert.Equal(s.T(), *experiment.ID, resp.ID)
+	assert.Equal(s.T(), experiment.Name, resp.Name)
+	assert.Equal(s.T(), experiment.LifecycleStage == models.LifecycleStageDeleted, resp.Archived)
```

Comment:
won't it be enough to just check `assert.Equal(s.T(), false, resp.Archved)` ?
also can you please check other properties: `description, creation_time`. I know that description right now is `nil` but it is nice to catch this moment when it will be changed.